lib/msal-node/src/utils/RetryTracker.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Logger } from "@azure/msal-common";
+
+export class RetryTracker {
+    private logger: Logger;
+    private retryLimit: number;
+    private retryWaitTimeMilli: number;
+
+    private _numRetried: number;
```

Comment:
inconsistent naming - all private fields should be named in the same way. 
Also, they can probably be readonly?